WebRoot/templates/calendar/Appointment.template

```diff
@@ -1,219 +1,329 @@
 <!-- NOTE: The columns are defined in a separate template so that they -->
 <!-- NOTE: can be overridden more easily by locale. For example, German -->
 <!-- NOTE: may need more room for the picker and label columns. -->
-<template id='calendar.Appointment#ComposeViewColumns'>
-    <colgroup>
-        <col width="80">
-        <col>
-        <col width="90">
-        <col width="130">
-        <col width="100">
-    </colgroup>
+<template id='calendar.Appointment#ComposeViewRowColumns'>
+	<colgroup>
+		<col width='<$= (AjxEnv.isIE || AjxEnv.isModernIE) ? "75px" : "1%" $>' style="white-space:nowrap;"/>
```

Comment:
is IE not considering percentage width for column?

Reply:
For this case, it was giving issue in IE (maybe because of nested tables), so I had to added that condition for IE only.